test/test_crosstalk.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+import numpy as np
+import tensorflow as tf
+
+from c3.generator.devices import Crosstalk
+from c3.generator.generator import Generator
+from c3.c3objs import Quantity as Qty
+from c3.parametermap import ParameterMap
+
+
+xtalk = Crosstalk(
```

Comment:
In general it is advisable to keep testing data and testing code decoupled. For example, how we have configs and jsons in the `test/` directory which is the input system data later used in the tests. The `pytest` way of doing this is to use `fixtures` and I think the `test_crosstalk()` could use some fixtures, e.g,  for the `crosstalk_matrix` etc. We currently use fixtures in the qiskit testing module, so you can check that for ref.

Reply:
done